deeplearning4j-keras/src/main/java/org/deeplearning4j/keras/DeepLearning4jEntryPoint.java

```diff
@@ -25,14 +23,15 @@ public void fit(EntryPointFitParameters entryPointFitParameters) throws Exceptio
         try {
             MultiLayerNetwork multiLayerNetwork = neuralNetworkReader.readNeuralNetwork(entryPointFitParameters);
 
```

Comment:
This is the Keras sequential model, no? :) 
Lets keep this PR simple - I'll add the ComputationGraph as a separate one (want to unblock myself to move on with the Python stuff)